packages/neo-one-client-core/src/user/connectRemoteUserAccountProvider.ts

```diff
@@ -0,0 +1,98 @@
+// tslint:disable only-arrow-functions
+import { Block, GetOptions, RawAction, TransactionReceipt, UserAccountProvider } from '@neo-one/client-common';
+import { Endpoint, Message } from './messageTypes';
+import { deserialize, serialize } from './serializeUtils';
+
+export interface ConnectRemoteUserAccountProviderOptions {
+  readonly endpoint: Endpoint;
+  readonly userAccountProvider: UserAccountProvider;
+}
+
+export const connectRemoteUserAccountProvider = ({
```

Comment:
This should return a function that when called disposes everything that was setup here (i.e. the event listeners and observable subscriptions)